config/routes.rb

```diff
@@ -9,6 +9,13 @@
     resources :comments, only: [:create]
     resources :likes, only: [:create, :destroy]
   end
+  resources :friendships, only: [:create, :destroy, :approve]
+  post '/addfriend',  to:'friendships#create'
+  get '/addfriend',  to:'friendships#create'
+  get '/unfriend', to:'friendships#destroy'
+  post '/unfriend', to:'friendships#destroy'
 
+  get '/approve_friendship', to:'friendships#approve'
+  post '/approve_friendship', to:'friendships#approve'
```

Comment:
We are asked to address these routes from the previous TSE :+1: Let us avoid repeating controller actions